WordPress/src/test/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModelTest.kt

```diff
@@ -113,20 +113,21 @@ class SiteCreationDomainsViewModelTest {
     }
 
     private fun <T> testWithErrorResponse(
```

Comment:
Thank you for sharing your thoughts on this @mkevins !

It's great and I really encourage you to keep sharing small ideas that may feel not important when they come up, there were already many times I had a similar idea in my mind. Finding out that you (or someone else) shares the same thoughts encourages me to take action on the rather abstract thought that I wouldn't otherwise.

> but just noting this, since reuse of such a function can have it grow more elaborate over time. I wonder if, at the cost of being more verbose (and less DRY), it'd be more readable over time to recreate the stubs for each test 🤔
I appreciate finding out that I'm not the only exercising this point of view 🙇